build.py

```diff
@@ -162,10 +162,10 @@ def main(args) :
         print('Run command requires a java class to run.')
```

Comment:
This script is designed for Python 2.x, which does not require parentheses. If you look, there's other instances in the build file which do not use parentheses. 

Since the change is unrelated to this pull request, let's revert it to keep things lean.